datadog/dogstatsd/base.py

```diff
@@ -160,10 +173,12 @@ def __enter__(self):
 
         def __exit__(self, type, value, traceback):
             # Report the elapsed time of the context manager.
-            self.statsd.timing(self.metric, time() - self.start,
-                               self.tags, self.sample_rate)
+            elapsed = time() - self.start
+            use_ms = self.statsd.use_ms or self.use_ms
```

Comment:
I'm unsure this really makes sense, someone could declare a statsd object with `use_ms` as `True`, then explicitly make a `_TimedContextManagerDecorator` with `use_ms` as `False` and this would resort to using milliseconds...is that expected?

Does this object even need this property or can it just pull it off its `statsd` object?